### PR TITLE
slirp: enable seccomp filter

### PR DESCRIPTION
**Pull request by giuseppe**
add a check for --enable-seccomp support in slirp4netns.  If it is
supported, always enable it.

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
NICE work
LGTM

**Comment by rhatdan** [comment]
@baude @mheon @vrothberg @TomSweeneyRedHat @QiWang19 @jwhonce PTAL

File: libpod/networking_linux.go [hunk 1]
```diff
@@ -154,13 +154,25 @@ func (r *Runtime) createNetNS(ctr *Container) (n ns.NetNS, q []*cnitypes.Result,
...
+func checkSlirpFlags(path string) (*slirpFeatures, error) {
 	cmd := exec.Command(path, "--help")
 	out, err := cmd.CombinedOutput()
 	if err != nil {
-		return false, false, false, errors.Wrapf(err, "slirp4netns %q", out)
-	}
-	return strings.Contains(string(out), "--disable-host-loopback"), strings.Contains(string(out), "--mtu"), strings.Contains(string(out), "--enable-sandbox"), nil
+		return nil, errors.Wrapf(err, "slirp4netns %q", out)
+	}
+	return &slirpFeatures{
```
Comment (TomSweeneyRedHat) [inline libpod/networking_linux.go]: I just wonder if a log.debug statement here would be helpful to let us know slirp had been automatically enabled?

**Comment by TomSweeneyRedHat** [comment]
One question for consideration, otherwise LGTM

**Comment by mheon** [comment]
LGTM

**Pull request merged**